### DISCUSSION TITLE
Accessing loading from js

### DISCUSSION THREAD

**Discussion by abdosaeedelhassan** [livewire/livewire#1306] [4. General]
is there away to know about now is loading or not from js?

**Comment by mblarsen** [comment]
You mean if there is a JS equivalent to the following?

```html
<div wire:loading>Loading...</div>
```

**Reply by abdosaeedelhassan** [reply]
Yes please, something like window.livewire.loading or any existing event that fired until data rendered on any operation like search or delete on big size of data
